### PR TITLE
Fixes Battlecruiser Turrets and Improves Turret's Faction Capabilities

### LINKED ISSUE #35425: Turrets on the battlecruiser shuttle now target regular borgs
Issue reported from  Round ID: 83231 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Reporting client version: 511

what it says on the tin

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
> faction = "turret"

fixes #35425

### CODE REVIEW

**Review by ShizCalev — COMMENTED** [review]
.

**Comment by KorPhaeron** [comment]
Should be neutral so it doesnt go nuts and kill Ian

**Comment by coiax** [comment]
What _are_ they supposed to shoot at?

**Comment by ExcessiveUseOfCobblestone** [comment]
> Should be neutral so it doesnt go nuts and kill Ian

I think turrets aren't like mob factions where it's a list.

**Comment by ExcessiveUseOfCobblestone** [comment]
no it's not give me one second

**Comment by ExcessiveUseOfCobblestone** [comment]
epic I edited my master instead

**Comment by ExcessiveUseOfCobblestone** [comment]
gbp NOW!

File: code/game/machinery/porta_turret/portable_turret.dm [hunk 1]
```diff
@@ -488,9 +488,11 @@
 
 
 /obj/machinery/porta_turret/proc/in_faction(mob/target)
-	if(!(faction in target.faction))
-		return 0
-	return 1
+	for(var/faction1 in faction)
+		if(faction1 in target.faction)
+			return TRUE
+		continue
```
Comment (Cyberboss) [inline code/game/machinery/porta_turret/portable_turret.dm]: Remove this continue

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by ShizCalev — CHANGES_REQUESTED** [review]
update the maps for the change to the faction var.
image

**Comment by ExcessiveUseOfCobblestone** [comment]
> var editing within maps when they're a pain to mess with
wtffffffff

**Comment by ExcessiveUseOfCobblestone** [comment]
I don't have python on this computer who do I need to smooch to do this simple var edits (or better yet make a subtype and replace those turrets ;) )

**Comment by ShizCalev** [comment]
@ExcessiveUseOfCobblestone you don't need python to do this, you can literally open the map up in notepad and replace it the var with `faction = list("pirate)`

They're also all different subtypes of turret, so there wouldn't be a one-size-fits-all in this case

**Comment by ExcessiveUseOfCobblestone** [comment]
@ShizCalev Snowdin must have been updated recently cuz it no longer has pirate faction.

**Pull request merged**